vectorizers/token_cooccurrence_vectorizer.py

```diff
@@ -543,7 +531,7 @@ def sequence_multi_skip_grams(
             array_to_tuple=array_to_tuple,
         )
     else:
-        if document_context:
+        if document_context == True:
```

Comment:
```suggestion
        if document_context:
```

If it is a boolean you shouldn't need to check equality to a boolean...